Add dblp ID

hauschke opened the pull request:
**[VIVO-Ontology GitHub issue](https://github.com/vivo-ontologies/vivo-ontology/issues)**: https://github.com/vivo-ontologies/vivo-ontology/issues/75

* Other relevant links (mailing list discussion, related pull requests, etc.)

# What does this pull request do?

Adds a new `rdfs:subPropertyOf http://vivoweb.org/ontology/core#identifier"` for person's IDs from https://dblp.uni-trier.de, an important bibliography for computer sciences with open bibliographic metadata.

# What's new?
Adds the property.

# Additional notes:
* Requires documentation to be updated

# Interested parties
@vivo-ontologies/team-members

hauschke committed:
add dblp ID

rdfs:subPropertyOf http://vivoweb.org/ontology/core#identifier"

Zack-83 commented on vivo.owl:
```diff
@@ -4800,7 +4799,14 @@ Definition source: https://ror.org/about/</obo:IAO_0000112>
 
+    <owl:DatatypeProperty rdf:about="http://vivoweb.org/ontology/core#dblpId">
+        <rdfs:label xml:lang="en">dblp ID</rdfs:label>
+        <obo:IAO_0000112 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">The dblp computer science bibliography provides open bibliographic information on major computer science journals and proceedings. Definition source: https://dblp.org/. The DBLP ID uniquely identifies researchers in the DBLP.</obo:IAO_0000112>
```
Maybe also here report the source with the `rdfs:isDefinedBy` notation, like in https://github.com/vivo-ontologies/vivo-ontology/blob/66f43cd48dde49fa34032e74a07525ca62ebec97/vivo.owl#L4800 ?  
Actually for coherence it should be implemented the same way also for the previuos 4000 lines...

hauschke committed:
Updated the dblp Person Key

Switched from dblp ID to dblp Person key in alignment with the dblp FAQ. Used the new annotation guidelines.

hauschke committed:
Mistake in URI

Zack-83 reviewed (changes requested):
Please update lines 4810-4815

hauschke committed:
Merge branch 'master' into add-dblp-id

hauschke commented on vivo.owl:
```diff
@@ -4800,7 +4799,24 @@ Definition source: https://ror.org/about/</obo:IAO_0000112>
+        <obo:IAO_0000115 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">In many ways, persons are identified in dblp on basis of the persons name string (optionally followed by a four digit number in case of several authors in dblp with the same name). That name can be found prominently in the grey bar at the top of a person's page. However, there is also a (more cryptic) unique identifie ...
+        <obo:IAO_0000118 xml:lang="en">dblp ID</obo:IAO_0000118>
+        <obo:IAO_0000119 rdf:resource="https://dblp.org/faq/5210253.html"/>           
+    </owl:DatatypeProperty>
```
It could cause problems to use <--> inside XML tags in obo:IAO_0000112. Thus, I did not include this change request.

In line 4815 I kept `<obo:IAO_0000119 rdf:resource="https://dblp.org/faq/5210253.html"/>` in alignment with the other PRs. 

I included your rdfs:comment.

Zack-83 replied:
Fine. Please see my new attempt

hauschke committed:
Changed dblpPersonKey property to new style

Zack-83 commented on vivo.owl:
```diff
@@ -4485,6 +4485,23 @@ modern society using the world of Star trek. Los Angeles Times, March
+        <rdfs:domain rdf:resource="http://xmlns.com/foaf/0.1/Person"/>    
+        <obo:IAO_0000111 xml:lang="en">dblp Person Key</obo:IAO_0000111>
+        <rdfs:comment xml:lang="en">A dblp person key is a code used to uniquely identify a scientist by the "dblp computer science bibliography", a joint service of Schloss Dagstuhl - Leibniz Center for Informatics and the University of Trier. The dblp person key, combined with the URI prefix, yields the resolvable URI of the entry. The key itself is an alphanumeric string composed of digits, le ...
+        <obo:IAO_0000112 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A dblp Person Key identifies a unique person in dblp such as https://dblp.org/pid/53/6062.html for Michael Conlon.</obo:IAO_0000112>
+        <vitro:exampleAnnot rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A dblp Person Key identifies a unique person in dblp such as https://dblp.org/pid/53/6062.html for Michael Conlon.</vitro:exampleAnnot>
+        <obo:IAO_0000115 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">In many ways, persons are identified in dblp on basis of the persons name string (optionally followed by a four digit number in case of several authors in dblp with the same name). That name can be found prominently in the grey bar at the top of a person's page. However, there is also a (more cryptic) unique identifie ...
+        <obo:IAO_0000118 xml:lang="en">dblp ID</obo:IAO_0000118>
+        <obo:IAO_0000119 rdf:resource="https://dblp.org/faq/5210253.html"/>
```
Not sure about the syntaxe OF the example and IN the example
```suggestion
        <obo:IAO_0000112 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Key: 53/6062 == URI: https://dblp.org/pid/53/6062.html == Name: Michael Conlon.</obo:IAO_0000112>
        <vitro:exampleAnnot rdf:datatype="http://www.w3.org/2001/XMLSchema#string">ex:articleXYZ dcterms:creator dblp:53/6062 . 
orcid:0000-0002-1304-8447 vivo:dblpPersonKey dblp:53/6062 .</vitro:exampleAnnot>
        <obo:IAO_0000115 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">In many ways, persons are identified in dblp on basis of the persons name string (optionally followed by a four digit number in case of several authors in dblp with the same name). That name can be found prominently in the grey bar at the top of a person's page. However, there is also a (more cryptic) unique identifier associated with a dblp person record. You can find this key listed in the "export" drop down menu right next to the name. The key can also be found as the "key" attribute of the "person" element in a persons XML export.</obo:IAO_0000115>
        <obo:IAO_0000118 xml:lang="en">dblp ID</obo:IAO_0000118>
        <obo:IAO_0000119 rdf:resource="https://dblp.org/faq/5210253.html"/>
```
